client/src/AppHeader/AppHeader.jsx

```diff
@@ -0,0 +1,17 @@
+// @flow
+
+import React from 'react';
+import { Input, Layout } from 'antd';
+import './AppHeader.scss';
+
+const { Search } = Input;
+
+export const AppHeader = () => (<Layout.Header className="app-header">
+  <h1 className="main-title">NeatEat</h1>
+  <h3 className="secondary-title">Never get hungry again.</h3>
+  <Search
+    placeholder="Find a restaurant..."
+    size="large"
+    className="search-restaurant-input"
+  />
+</Layout.Header>);
```

Comment:
the new line before the )
and let make sure that the linter is configured correctly in the IDE